WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.java

```diff
@@ -150,52 +130,64 @@ public void onSaveInstanceState(Bundle outState) {
 
     @Override
     public void onPositiveClicked(@NotNull String instanceTag) {
-        if (mPostList != null) {
-            mPostList.onPositiveClickedForBasicDialog(instanceTag);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onPositiveClickedForBasicDialog(instanceTag);
         }
     }
 
     @Override
     public void onNegativeClicked(@NotNull String instanceTag) {
-        if (mPostList != null) {
-            mPostList.onNegativeClickedForBasicDialog(instanceTag);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onNegativeClickedForBasicDialog(instanceTag);
         }
     }
 
     @Override
     public void onDismissByOutsideTouch(@NotNull String instanceTag) {
-        if (mPostList != null) {
-            mPostList.onDismissByOutsideTouchForBasicDialog(instanceTag);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onDismissByOutsideTouchForBasicDialog(instanceTag);
         }
     }
 
-    // GutenbergWarningDialogClickInterface Callbacks
+// GutenbergWarningDialogClickInterface Callbacks
```

Comment:
Nitpick: It looks like this comment was shifted, can we indent it?

Reply:
Fixed in [39a5ad4](https://github.com/wordpress-mobile/WordPress-Android/pull/9226/commits/39a5ad4ce6e3530c096b11a24ae5fcc5e804f5a4)